migrace/2024-07-04_kody-typu-akci.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/** @var \Godric\DbMigrations\Migration $this */
+
+$this->q(<<<SQL
+alter table akce_typy
+    add kod_typu varchar(20) null comment 'kód pro identifikaci například v rozpočtovém reportu'
+SQL
+);
+
+$this->q(<<<SQL
+UPDATE akce_typy t
+SET t.kod_typu = 'Larp'
+WHERE t.id_typu = 2
+SQL
+);
+
+$this->q(<<<SQL
+UPDATE akce_typy t
+SET t.kod_typu = 'AH'
+WHERE t.id_typu = 7
+SQL
+);
+
+$this->q(<<<SQL
+UPDATE akce_typy t
+SET t.kod_typu = 'RPG'
+WHERE t.id_typu = 4
+SQL
+);
+
+$this->q(<<<SQL
+UPDATE akce_typy t
+SET t.kod_typu = 'LKD'
+WHERE t.id_typu = 8
+SQL
+);
+
+$this->q(<<<SQL
+UPDATE akce_typy t
+SET t.kod_typu = 'WG'
+WHERE t.id_typu = 6
+SQL
+);
+
+$this->q(<<<SQL
+UPDATE akce_typy t
+SET t.kod_typu = 'Epic'
+WHERE t.id_typu = 11
+SQL
+);
+
+$this->q(<<<SQL
+UPDATE akce_typy t
+SET t.kod_typu = 'Turn'
+WHERE t.id_typu = 1
+SQL
+);
+
+$this->q(<<<SQL
+UPDATE akce_typy t
+SET t.kod_typu = 'DrD'
+WHERE t.id_typu = 9
+SQL
+);
+
+$this->q(<<<SQL
+UPDATE akce_typy t
+SET t.kod_typu = 'Pred'
```

Comment:
Ty kódy jsou docela nečitelné, to vymyslel Sirien co ? :D

Já bych klidně použil celá slova, například `PREDNASKY`.

Ovšem pokud ty kódy potřebujeme jenom pro jeden report a ještě k tomu velmi specifický, tak bych je klidně dal až do SQL toho reportu a nepřidával bych je natvrdo do sloupce. Ono se to pak sejde a jedna tabulka má tolik sloupců s tolika zvláštními hodnotami, že je to nepřehledné.

Ale je to jenom k zamyšlení, klidně to takhle nech.

Reply:
Jojo, jsou sirienovy...
Já jsem uvažoval tak, že pokud je chce jako standardizované zkratky jednotlivých sekcí, tak je můžeme rovnou ukládat u těch sekcí a aspoň už budou v tabulce až bude potřeba nějaký nový report nebo cokoliv

Reply:
Po domluvě se Sirienem změňme prosím `Pred` na `Prednasky`.

Čekal jsem těhle zkratek víc, ale zjevně to byly jen ty přednášky, takže takhle by to bylo všechno.